index.py

```diff
@@ -0,0 +1,91 @@
+from telegram.ext import (
+    Updater,
+    Filters,
+    MessageHandler,
+    ConversationHandler,
+    CommandHandler,
+)
+import os
+from invalidmsg import wrongOption
+from adminpanel import adminpanel, adminoptions, addadmin, removeadmin
+from dotenv import load_dotenv
+from unknownHandler import unknown
+from startHandler import start
+from memeHandler import meme
+from jokeHandler import joke
+from coronaHandler import countrySelection, dateSelection, coronaupdates
+from addmykey import askkey, addkey
+from shell import shellsession, runcommand
+from about import about_website
+from chatid import getChatID
+from smshandler import asknum, askmsg, sendsms
+
+
+config = ".env" if os.path.exists(".env") else "sample.env"
+load_dotenv(dotenv_path=config)
+token = os.getenv("TOKEN")
+updater = Updater(token=token, use_context=True)
+dispatcher = updater.dispatcher
+joke_handler = CommandHandler("joke", joke)
+dispatcher.add_handler(joke_handler)
+cID = CommandHandler("getchatid", getChatID)
+dispatcher.add_handler(cID)
+corona_states = {
+    0: [MessageHandler(Filters.text, dateSelection)],
+    1: [MessageHandler(Filters.text, coronaupdates)],
+}
+corona_handler = ConversationHandler(
```

Comment:
For each of the handler, create `ConservationHandler` inside the handler file for that handler, and then import just the respective `ConservationHandler` from there into index.py. That we, each handler file is consistent, its exports a handler file which is imported here, and we don't have to import internal functions from the handler, that is encapsulated inside that file. 

Just to be clear, so in index.py you would instead just do
```python
# Pseudocode index.py
import corona_handler, meme_handler from handlers
.....
....
dispatcher.add_handler(corona_handler)
dispatcher.add_handler(meme_handler)
...
```

Reply:
[Made a few changes](https://github.com/alias-rahil/CSI-BroBot/commit/2493ef964ce666752cdee6d2f7ef903cf72961d7), please review.